workflow/pred_temperature_ridge.ipynb

```diff
@@ -102,92 +114,79 @@
        ")"
       ]
      },
-     "execution_count": 3,
+     "execution_count": 51,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "# check calendar\n",
+    "# Check calendar\n",
     "calendar"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Load test data SST and (clustered) T2M\n",
-    "We use 63 years (1959-2021) of data."
+    "### 2 - Load datasets\n",
+    "\n",
+    "Load Sea Surface Temperature (SST) and (clustered) 2m air temperature data (T2M). Here we use data from 1959-2021 (63 years total). "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 52,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "('t2m_daily_1959-2021_2deg_clustered_226_300E_30_70N.nc',\n",
-       " <http.client.HTTPMessage at 0x121aed700>)"
-      ]
-     },
-     "execution_count": 4,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "# URL of the dataset from zenodo\n",
-    "sst_url = \"https://zenodo.org/record/8186914/files/sst_daily_1959-2021_5deg_Pacific_175_240E_25_50N.nc\"\n",
-    "t2m_url = \"https://zenodo.org/record/8186914/files/t2m_daily_1959-2021_2deg_clustered_226_300E_30_70N.nc\"\n",
-    "sst_field = \"sst_daily_1959-2021_5deg_Pacific_175_240E_25_50N.nc\"\n",
-    "t2m_field = \"t2m_daily_1959-2021_2deg_clustered_226_300E_30_70N.nc\"\n",
-    "\n",
-    "urllib.request.urlretrieve(sst_url, sst_field)\n",
-    "urllib.request.urlretrieve(t2m_url, t2m_field)"
+    "def load_data(url, field, var):\n",
+    "    # Download and load dataset\n",
+    "    url_field = url + field\n",
+    "    urllib.request.urlretrieve(url_field, field)\n",
+    "    dataset = xr.open_dataset(field)\n",
+    "    dataset = dataset[var] - 273.15\n",
+    "    return dataset "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 53,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# load data\n",
-    "precursor_field = xr.open_dataset(sst_field)\n",
-    "target_field = xr.open_dataset(t2m_field)"
+    "# URL of the dataset from zenodo\n",
+    "url = \"https://zenodo.org/record/8186914/files/\"\n",
+    "\n",
+    "sst_field = \"sst_daily_1959-2021_5deg_Pacific_175_240E_25_50N.nc\"\n",
+    "t2m_field = \"t2m_daily_1959-2021_2deg_clustered_226_300E_30_70N.nc\""
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 54,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Convert Klevin to Celcius\n",
-    "precursor_field[\"sst\"] = precursor_field[\"sst\"] - 273.15\n",
-    "target_field[\"t2m\"] = target_field[\"t2m\"] - 273.15"
+    "precursor_field = load_data(url, sst_field, \"sst\")\n",
+    "target_field = load_data(url, t2m_field, \"t2m\").sel(cluster=3)"
```

Comment:
Add a comment here that cluster 3 represents the southwestern USA?

Reply:
Thanks also added a reference to the Pacific for the SST